Doc/library/decimal.rst

```diff
@@ -790,7 +790,7 @@ Decimal objects
       the current thread's context is used.
 
       An error is returned whenever the resulting exponent is greater than
-      :attr:`Emax` or less than :attr:`Etiny`.
+      :attr:`Emax` or less than :meth:`Etiny`.
```

Comment:
Urgh. Did we _really_ fail to make `Etiny` a property rather than a method? 😞

Reply:
Yup, had the same thought myself but I double-checked and it sure is :(